scipy/stats/_continuous_distns.py

```diff
@@ -9033,6 +9034,101 @@ def _munp(self, n, b):
+    which are well known to be just cardinal B-splines of degree :math:`n-1` 
+    with knots evenly spaced from :math:`1` to :math:`n`.
+
+    The Bates distribution is simply the Irwin-Hall distribution scaled 
+    by :math:`1/n`: ``bates = irwinhall(n, name="bates", scale=1/n)``
+
+    
+    %(after_notes)s
+
+    References
+    ----------
+    .. [1] HALL PHILIP, "THE DISTRIBUTION OF MEANS FOR SAMPLES OF SIZE N DRAWN 
+            FROM A POPULATION IN WHICH THE VARIATE TAKES VALUES BETWEEN 0 AND 1, 
+            ALL SUCH VALUES BEING EQUALLY PROBABLE", 
+            Biometrika, Volume 19, Issue 3-4, December 1927, Pages 240-244, 
+            https://doi.org/10.1093/biomet/19.3-4.240
+    .. [2] J. O. IRWIN, "ON THE FREQUENCY DISTRIBUTION OF THE MEANS OF SAMPLES 
+            FROM A POPULATION HAVING ANY LAW OF FREQUENCY WITH FINITE MOMENTS, 
+            WITH SPECIAL REFERENCE TO PEARSON'S TYPE II", 
+            Biometrika, Volume 19, Issue 3-4, December 1927, Pages 225-239, 
+            https://doi.org/10.1093/biomet/19.3-4.225
+    .. [3] K. Buchanan, T. Adeyemi, C. Flores-Molina, S. Wheeland and D. Overturf, 
+            "Sidelobe behavior and bandwidth characteristics 
+            of distributed antenna arrays," 
+            2018 United States National Committee of 
+            URSI National Radio Science Meeting (USNC-URSI NRSM), 
+            Boulder, CO, USA, 2018, pp. 1-2.
+            https://www.usnc-ursi-archive.org/nrsm/2018/papers/B15-9.pdf
+
+    %(example)s
+    """
+
+    def _argcheck(self, n):
+        return (n > 0) & _isintegral(n) & np.isrealobj(n)
+    
+    def _get_support(self, n):
+        return 0, n 
+    
+    def _shape_info(self):
+        return [_ShapeInfo("n", True, (1, np.inf), (True, False))]
+    
+    def _munp(self, n, t):
+        return (sc.expm1(t)/t)**n
+        
+    @staticmethod
+    def _cardbspl(n):
+        t = np.arange(n+1) 
+        return BSpline.basis_element(t)
+
+    def _pdf(self, x, n):
+        def vpdf(x, n):
+            return self._cardbspl(n)(x)
+        return np.vectorize(vpdf, otypes=[np.float64])(x, n)
+    
+    def _cdf(self, x, n):
+        def vcdf(x, n):
+            return self._cardbspl(n).antiderivative()(x)
+        return np.vectorize(vcdf, otypes=[np.float64])(x, n)
+    
+    def _rvs(self, n, size=None, random_state=None, *args):
+        @_vectorize_rvs_over_shapes
+        def _rvs1(n, size=None, random_state=None):
+            if not self._argcheck(n):
+                raise ValueError("n must be a positive integer")
+            n = np.floor(n).astype(int)
+            if size is None:
+                return np.sum(random_state.uniform(size=n)).item()
+            ret = random_state.uniform(size=(n, *size)).sum(axis=0)
+            return ret
+        return _rvs1(n, size=size, random_state=random_state)
+    
+    def _stats(self, n):
+        # mgf = ((exp(t) - 1)/t)**n
+        # m'th derivative follows from the generalized Leibniz rule
+
```

Comment:
To be honest it derives trivially from the definition as the sum of n independent uniform rvs which implies that the MGF is just the uniform one to the nth power

Reply:
We all have different backgrounds, and so what is trivial to some (https://github.com/scipy/scipy/pull/20481#discussion_r1582587592) may not be clear to all. Thanks for extending the comment, though. I'll go ahead and add the direct reference for those who would prefer to avoid the derivation.